src/resources/admin-notices.js

```diff
@@ -1,13 +1,36 @@
-jQuery(document).ready(function ($) {
-    const {dispatch} = wp.data;
-    const dismissButtons = $('.notice-dismiss');
+(function ($, dispatch, document) {
+    // Set up the package functions using the namespace provided by the script tag.
+    const currentScript = typeof document.currentScript !== 'undefined' ? document.currentScript : document.scripts[document.scripts.length - 1];
+    const namespace = currentScript.getAttribute('data-stellarwp-namespace');
```

Comment:
```suggestion
    const namespace = currentScript.getAttribute('data-stellarwp-namespace');
```
This concept and data attribute will be standardized across other libs? The naming implies that and of course this will be a common issue to handle.

Reply:
Yeah, I stole this idea from another Stellar package: https://github.com/stellarwp/installer/blob/9c2774a1fcc9724839c7f009784420ed71831fde/src/assets/js/installer.js

It's a good way to go about this!